decidim-core/app/packs/src/decidim/redesigned_a11y.js

```diff
@@ -49,6 +49,13 @@ const createDropdown = (component) => {
     component.addEventListener("click", () => setTimeout(() => document.getElementById(autofocus).focus(), 0));
   }
 
+  const scrollToMenu = component.dataset.scrollToMenu === "true";
+  if (scrollToMenu) {
+    // Auto scroll to show the menu on the viewport
+    // @see: https://github.com/decidim/decidim/issues/11307
```

Comment:
No need to add the issue number with the explanation. If we do that for every feature we would have literally hundreds of comments with issues. 
This should be done just when the explanation of the feature/problem is too big for a comment, but the other comment (`Auto scroll to show the menu on the viewport`) is really clear as is
```suggestion
```